.github/workflows/create-draft-release.yml

```diff
@@ -0,0 +1,53 @@
+name: "Create Draft Release"
+
+env:
+  BUILDDIR: "build"
+  DEVELOPER_DIR: "/Applications/Xcode_12.5.1.app/Contents/Developer"
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to use for this release. Please use https://semver.org.'
+        required: true
+
+concurrency: 
+  group: publish-release-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release:
+    name: "Publish binaries for release"
+    runs-on: macos-11
+
+    steps:
+      - name: "Checkout sources"
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+          fetch-depth: 0
+
+      - name: "Build release distribution"
+        run: make release
```

Comment:
We may not need to create a tag yet, but we need to provide make-release-package.sh the tag name (see `$latest_git_tag` usage)

Reply:
As part of this workflow, we don't want to run release-move-tag.sh or any of the other commands except for the xcodebuild one in `make release` to my understanding.

Reply:
release-move-tag.sh also interactively asks the user a question so we can't allow that to happen in CI

Reply:
On second thought, I think we should still run release-move-tag.sh - but we should make it automatically make the commit / move the tag and not interactively ask the user.

Reply:
This is addressed by 3e943bf5f7b45286ab50353032561748f8c33079.